### PR TITLE
[LV] Add Latvian binary sensor get state translations

### PR DESCRIPTION
**Pull request by klejejs**
Added sentences and tests for Latvian language binary sensor get state functionality.

The PR and translations are heavily inspired by English language implementation.

## Summary by CodeRabbit

- **New Features**
  - Enhanced the `HassGetState` intent in Latvian (`lv`) to include new response variations (`one_yesno`, `any`, `all`, `which`, `how_many`) based on query context.
  - Introduced comprehensive intents for querying states of various binary sensors (battery, carbon monoxide, motion, etc.) in home automation.

- **Bug Fixes**
  - Corrected and updated binary sensor state values to ensure accurate responses for various conditions (e.g., battery, connectivity, door status).

- **Tests**
  - Added extensive test cases in Latvian for different binary sensor states to validate `HassGetState` intents and responses.

### CODE REVIEW

File: sentences/lv/_common.yaml [hunk 1]
```diff
@@ -44,6 +44,171 @@ lists:
...
+    values:
+      - in: "(detektēt(s|a|i)|aktiv[iz]ēt(s|a|i)|ieslēgt(s|a|i|as)|[ir ]kustība)"
+        out: "on"
+      - in: "(nav detektēt(s|a|i)|deaktiv[iz]ēt(s|a|i)|izslēgt(s|a|i|as)|nav kustība[s])"
+        out: "off"
+  bs_occupancy_states:
+    values:
+      - in: "([ir ](kād(s|a|i|as)|cilvēk(s|i)|[māj]dzīvniek(s|i)|aizņemt[īb](s|a|i|as)|aktiv[iz]ēt(s|a|i)|ieslēgt(s|a|i|as)))"
+        out: "on"
+      - in: "(nav (nevien(s|a)|cilvēk(s|i)|[māj]dzīvniek(s|i)|aizņemt[īb](s|a|i|as)|aktiv[iz]ēt(s|a|i)|ieslēgt(s|a|i|as)))"
```
[the remark this answers is not in the document]
Comment (klejejs) [reply sentences/lv/_common.yaml:131]: Don't think space can be removed here.

File: sentences/lv/_common.yaml [hunk 2]
```diff
@@ -44,6 +44,171 @@ lists:
...
+    values:
+      - in: "aiztikt(s|a|i|as)"
+        out: "on"
+      - in: "(nav |ne)aiztikt(s|a|i|as)"
+        out: "off"
+  bs_update_states:
+    values:
+      - in: "[(pieejam(s|i|ie)|var) ]atjaun(ot[(s|a|i)]|ojam(s|a|i)|inā(t[(s|a|i)]|jum(s|i|am)|(ms|ma|mi|mas)))"
+        out: "on"
+      - in: "(nav|nevar) [pieejam(s|i|ie) ]atjaun(ot(s|a|i)|ojam(s|a|i)|inā(t(s|a|i)|jum(s|i|am)))"
```
[the remark this answers is not in the document]
Comment (klejejs) [reply sentences/lv/_common.yaml:197]: Don't think space can be removed here.

File: sentences/lv/_common.yaml [hunk 3]
```diff
@@ -44,6 +44,171 @@ lists:
...
+    values:
+      - in: "(slapj(š|a|i)|applūd(is|usi|uši|ušas))"
+        out: "on"
+      - in: "saus(s|a|i)"
+        out: "off"
+  bs_motion_states:
+    values:
+      - in: "(detektēt(s|a|i)|aktiv[iz]ēt(s|a|i)|ieslēgt(s|a|i|as)|[ir ]kustība)"
+        out: "on"
+      - in: "(nav detektēt(s|a|i)|deaktiv[iz]ēt(s|a|i)|izslēgt(s|a|i|as)|nav kustība[s])"
```
[the remark this answers is not in the document]
Comment (klejejs) [reply sentences/lv/_common.yaml:125]: This needs `[s]` for matching.

File: sentences/lv/_common.yaml [hunk 4]
```diff
@@ -44,6 +44,171 @@ lists:
...
+        out: "off"
+  bs_tamper_states:
+    values:
+      - in: "aiztikt(s|a|i|as)"
+        out: "on"
+      - in: "(nav |ne)aiztikt(s|a|i|as)"
+        out: "off"
+  bs_update_states:
+    values:
+      - in: "[(pieejam(s|i|ie)|var) ]atjaun(ot[(s|a|i)]|ojam(s|a|i)|inā(t[(s|a|i)]|jum(s|i|am)|(ms|ma|mi|mas)))"
```
[the remark this answers is not in the document]
Comment (klejejs) [reply sentences/lv/_common.yaml:195]: Don't think space can be removed here.

File: sentences/lv/_common.yaml [hunk 5]
```diff
@@ -44,6 +44,171 @@ lists:
...
+        out: "off"
+  bs_moisture_states:
+    values:
+      - in: "(slapj(š|a|i)|applūd(is|usi|uši|ušas))"
+        out: "on"
+      - in: "saus(s|a|i)"
+        out: "off"
+  bs_motion_states:
+    values:
+      - in: "(detektēt(s|a|i)|aktiv[iz]ēt(s|a|i)|ieslēgt(s|a|i|as)|[ir ]kustība)"
```
[the remark this answers is not in the document]
Comment (klejejs) [reply sentences/lv/_common.yaml:123]: Space is needed only if `ir` is matched.

File: sentences/lv/binary_sensor_HassGetState.yaml [hunk 6]
```diff
@@ -0,0 +1,813 @@
...
+          - "vai <name> [<area>] [ir] {bs_battery_states:state}"
+        response: one_yesno
+        requires_context:
+          domain: binary_sensor
+          device_class: battery
+        slots:
+          domain: binary_sensor
+          device_class: battery
+      - sentences:
+          - "vai kāda [no] baterij(a|ām) [<area>] [ir] {bs_battery_states:state}"
```
[the remark this answers is not in the document]
Comment (klejejs) [reply sentences/lv/binary_sensor_HassGetState.yaml:17]: It does not make sense to break sentences into multiple lines.

**Comment by synesthesiam** [comment]
@makstech Does this look OK to you?

**Pull request merged**